app/src/main/java/edu/uc/app/parkfinder/service/ParkService.kt

```diff
@@ -10,18 +12,20 @@ import retrofit2.Response
 
 class ParkService {
 
-    fun fetchParks() : MutableLiveData<ArrayList<Park>> {
-        var _parks = MutableLiveData<ArrayList<Park>>()
+    internal fun fetchParks(): MutableLiveData<ArrayList<Park>> {
```

Comment:
visibility modifier should be internal https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces

internal means that any client inside this module who sees the declaring class sees its internal members.

Reply:
Good.